### PR TITLE
Add stale configuration

### PR DESCRIPTION
**Pull request by jpignata**
*Issue #, if available:* n/a

*Description of changes:* Uses the stale.yml from the SDK repo here to configure Stale bot.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by TrekSoft — APPROVED** [review]
Don't understand the function of a stale policy if we only mark them stale after 100 years but hey... :P

Cat10u3

**Comment by jpignata** [comment]
Yeah, not sure, I did not think about it too deeply. The functionality I'd like to port over is the "Closing Soon" behavior to act as something like a seven day auto-resolve.

File: .github/stale.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+staleLabel: Closing Soon
+# Comment to post when marking an issue as stale. Set to `false` to disable
+markComment: >
+  This issue has been automatically marked as stale because it has not had
+  recent activity. It will be closed if no further activity occurs. Thank you
+  for your contributions.
+# Comment to post when closing a stale issue. Set to `false` to disable
+closeComment: >
+  This issue has been automatically closed because of inactivity.
+  Please open a new issue if are still encountering problems.
```
Comment (richardmcclellan) [inline .github/stale.yml:16]:
```suggestion
  Please open a new issue if you are still encountering problems.
```

Comment (jpignata) [reply .github/stale.yml]: Guess I need to backport this. :P

**Review by richardmcclellan — APPROVED** [review]

**Pull request merged**